### ISSUE TITLE
patronictl pause/resume does not work with self-signed certificat

### ISSUE THREAD

**Issue by wilfriedroset** [zalando/patroni#786]
When using patronictl pause or resume action while having the rest api using a self-signed certificat, we cannot set the cluster in maintenance

```
patronictl -c config.yml pause
2018-08-20 16:32:39,289 - WARNING - Member patroni-99-node-1 is not accessible
2018-08-20 16:32:39,301 - WARNING - Member patroni-99-backup-1 is not accessible
2018-08-20 16:32:39,313 - WARNING - Member patroni-99-node-2 is not accessible
Error: Can not find accessible cluster member
```

This is related to  https://github.com/zalando/patroni/issues/785
patroni version 1.4.4 installed from pgdg repository, debian strech

**Comment by CyberDem0n** [comment]
Off course it is related.
patronictl can do a lot of different requests to Patroni API. So far you discovered only two.
There are also: `flush`, `reinit`, `restart` and `reload` and they are all will fail because of self-signed certificate, but I don't think that every command deserves a separate issue.

**Comment by wilfriedroset** [comment]
fixed by #790
